src/VSIX/VSIX.csproj

```diff
@@ -1,60 +1,62 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
+    <MinimumVisualStudioVersion>$(VisualStudioVersion)</MinimumVisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+    <VsixType>v3</VsixType>
```

Comment:
I'm guessing that VS reordered all these properties. Ideally, the commit wouldn't change the ordering; the idea was to have th generic/unchanged settings below, and only overrides/Npgsql-specific stuff above. Also, the ordering makes it harder to see what actually changed.

Reply:
OK, am fine with new orderings, no problem.